### PR TITLE
Remove deprecated types

### PR DESCRIPTION
**Pull request by kilativ-dotcom**

### CODE REVIEW

File: problem-solver/cxx/inferenceModule/generator/SolutionTreeGenerator.cpp [hunk 1]
```diff
@@ -62,21 +62,21 @@ ScAddr SolutionTreeGenerator::createSolutionNode(
...
     if (replacement.IsValid())
     {
-      ScAddr const & pair = ms_context->GenerateNode(ScType::NodeConst);
-      ms_context->GenerateConnector(ScType::EdgeAccessConstPosPerm, replacementsNode, pair);
+      ScAddr const & pair = ms_context->GenerateNode(ScType::ConstNode);
+      ms_context->GenerateConnector(ScType::ConstPermPosArc, replacementsNode, pair);
       GenerationUtils::generateRelationBetween(ms_context, pair, replacement, ScKeynodes::rrel_1);
       GenerationUtils::generateRelationBetween(ms_context, pair, variable, ScKeynodes::rrel_2);
-      ms_context->GenerateConnector(ScType::EdgeAccessConstPosTemp, variable, replacement);
+      ms_context->GenerateConnector(ScType::ConstTempPosArc, variable, replacement);
```
Comment (NikitaZotov) [inline problem-solver/cxx/inferenceModule/generator/SolutionTreeGenerator.cpp:79]: I think we can specify actuality here

Comment (NikitaZotov) [reply problem-solver/cxx/inferenceModule/generator/SolutionTreeGenerator.cpp:79]: Resee everywhere please

File: problem-solver/cxx/solutionModule/manager/DeleteSolutionManager.hpp [hunk 2]
```diff
@@ -28,7 +28,7 @@ class DeleteSolutionManager
 
   ScAddrList getListFromSet(ScAddr const & set) const;
 
-  void deleteEdges(ScAddr const & source, ScType const & edge, ScAddr const & target) const;
+  void deleteConnectors(ScAddr const & source, ScType const & connectorType, ScAddr const & target) const;
```
Comment (NikitaZotov) [inline problem-solver/cxx/solutionModule/manager/DeleteSolutionManager.hpp:31]:
```suggestion
  void eraseConnectors(ScAddr const & source, ScType const & connectorType, ScAddr const & target) const;
```

Comment (NikitaZotov) [reply problem-solver/cxx/solutionModule/manager/DeleteSolutionManager.hpp:31]: check everywhere such prefixes as create, generate, erase, delete, remove, get, search, find

**Review by NikitaZotov — CHANGES_REQUESTED** [review]

File: problem-solver/cxx/inferenceModule/generator/SolutionTreeGenerator.cpp (hunk 1 above)
Comment (kilativ-dotcom) [reply problem-solver/cxx/inferenceModule/generator/SolutionTreeGenerator.cpp:79]: I don't think so
for one formula this substitution is actual, but there is a lot of solution trees in the kb and variables may have different substitutions at every formula usage

**Review by NikitaZotov — APPROVED** [review]

**Pull request merged**